src/main/java/duke/FileParseException.java

```diff
@@ -0,0 +1,19 @@
+package duke;
+
+/**
+ * Represents an exception that occurs when the
```

Comment:
Perhaps you can consider editing your JavaDoc comment to the following for better phrasing: "Throws an Exception when the file format does not conform with the expected String representation of a Task." 😄